### DISCUSSION TITLE
Usermention in Checkins like on Twitter

### DISCUSSION THREAD

**Discussion by Oxbowatos** [Traewelling/traewelling#2045] [Ideas / Feature Requests]
My Idea is that it's possible to mention Users in Checkins and that it gets a link which redirects to the user. 
I always see how peoples write something like: "Driving with @Username & @Username2" or "Met @username in the metro", etc. 

So the Idea is that peoples can mention Users, the user that got mentioned gets a Notification like "You got mentioned in a checkin from @username" and maybe when there are too many users just make it possible to mention users that you follow or follow you.

**Comment by MrKrisKrisu** [comment]
closed with #2336
